mdoc/Mono.Documentation/Frameworks/FrameworkIndex.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+
+using Mono.Cecil;
+
+namespace Mono.Documentation
+{
+	public class FrameworkTypeEntry : IComparable<FrameworkTypeEntry>
```

Comment:
I'm also not at all sure why this needs to go into a `Frameworks` subdirectory...